posthog/queries/session_recordings/test/test_session_recording_list.py

```diff
@@ -69,43 +69,46 @@ def test_basic_query(self):
             self.assertEqual(session_recordings[1]["distinct_id"], "user")
             self.assertEqual(more_recordings_available, False)
 
-            # no event means no properties
-            self.assertEqual(session_recordings[0]["properties"], {})
-            self.assertEqual(session_recordings[1]["properties"], {})
-
         @freeze_time("2021-01-21T20:00:00.000Z")
-        def test_properties(self):
+        def test_metadata_list(self):
```

Comment:
Use @snapshot_clickhouse_queries for query-based tests - this way you don't need to manually write them into descriptions